File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://training-support.net")

    pageTitle = driver.title

    print("Title of the page: ", pageTitle)

    element = driver.find_element(By.LINK_TEXT, "About Us").click()

    print("New page title is : ", driver.title)

    driver.quit()

File: Selenium/Activities/Activity1.java
package Activity;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
	public static void main(String[] args) {
		
		WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net");
		
		System.out.println("Title of the web page:" + driver.getTitle());
		
		driver.findElement(By.linkText("About Us")).click();
				
		System.out.println("Title of new page: " + driver.getTitle());
		
		driver.quit();
		
		
	}

}
